Fabric.Authorization.AccessControl/src/app/modules/access-control/custom-group/custom-group.component.html

```diff
@@ -65,16 +67,25 @@ <h3>Selected</h3>
             <div class="table-container">
                 <table class="hc-table hc-table-borders">
                     <tbody>
-                        <tr *ngFor="let user of associatedPrincipals">
+                        <tr *ngFor="let user of associatedUsers">
```

Comment:
Would it be better to collapse these into an associatedPrincipals? I see that is how it was before, just curious if we ran into issues with that, because I think the member component works similarly.